cli/index.js

```diff
@@ -39,13 +43,354 @@ const signUp = (args) => {
+        .then((res) => {
+            console.log(res.data);
+
+            const FBIdToken = `Bearer ${res.data.token}`;
+            // TODO: create a file in the user's machine to store the auth token
+            console.log(chalk.green.bold("user guest created successfully ✔️"));
+        })
+        .catch((error) => {
+            console.log(chalk.red.bold(`Could not create aguest account -> code: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
+        });
+}
+
+//* Login 
+const login = (args) => {
+
+    const { password, username } = args;
+
+    const useUserData = {
+        password,
+        username,
+    };
+
+    console.log(chalk.yellow.bold("login..."));
+
+    axios
+        .post("/user/login", useUserData)
+        .then((res) => {
+            console.log(res.data);
+
+            const FBIdToken = `Bearer ${res.data.token}`;
+            // TODO: create a file in the user's machine to store the auth token
+            console.log(chalk.green.bold("login successfully ✔️"));
+        })
+        .catch((error) => {
+            console.log(chalk.red.bold(`Could not login -> code: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
+        });
+}
+
+// implement user/logout
+   {  console.log(chalk.yellow.bold("Logging out..."));
+
+    axios
+        .post("/user/logout")
+        .then((res) => {
+            console.log(res.data);
+
+            const FBIdToken = `Bearer ${res.data.token}`;
+            
+            console.log(chalk.green.bold(" successfully operation ✔️"));
+        })
+        .catch((error) => {
+            console.log(chalk.red.bold(`Could not log out: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
+        });
+
+    }
+//* recover 
+const recover = (args) => {
+
+    const { username, userId, email, newPassword, confirmPassword } = args;
+
+    const recoverUserData = {
+        username,
+        userId,
+        email,
+        newPassword,
+        confirmPassword,
+    };
+
+    console.log(chalk.yellow.bold("recovering..."));
+
+    axios
+        .put("/user/recover", recoverUserData)
+        .then((res) => {
+            console.log(res.data);
+
+            const FBIdToken = `Bearer ${res.data.token}`;
+            // TODO: create a file in the user's machine to store the auth token
+            console.log(chalk.green.bold("recover successfully ✔️"));
+        })
+        .catch((error) => {
+            console.log(chalk.red.bold(`Could not recover the account -> code: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
+        });
+}
+
+//* delete 
+const deleteuser = (args) => {
+
+    const { username } = args;
+
+    const deleteUserData = {
+        username,
+    };
+
+    console.log(chalk.yellow.bold("borrando..."));
+
+    axios
+        .delete('/user/${deleteUserData.username}')
+        .then((res) => {

```

Comment:
this is not going to work, have you tested this? In javascript when you want to include the value of the variable inside the string, you need to use backticks, and not single quotes: ``` ``  ```